lib/Dialect/HW/HWOps.cpp

```diff
@@ -261,14 +261,14 @@ HWModulePortAccessor::HWModulePortAccessor(Location loc,
                                            const ModulePortInfo &info,
                                            Region &bodyRegion)
     : info(info) {
-  inputArgs.resize(info.inputs.size());
+  inputArgs.resize(info.size_inputs());
   for (auto [i, barg] : llvm::enumerate(bodyRegion.getArguments())) {
-    inputIdx[info.inputs[i].name.str()] = i;
+    inputIdx[info.at(i).name.str()] = i;
```

Comment:
This should probably be `at_input` to be not so reliant on the internal ordering.

Reply:
Or even better, switch to using an iterator.